### DISCUSSION TITLE
wazuh Elasticsearch Utilities and osquery

### DISCUSSION THREAD

**Discussion by 011248163264** [Security-Onion-Solutions/securityonion#8164] [Unsupported Versions] [answered]
hi to all, i have a question regarding the utilities, is it recommended to install all utilities i.e. wazuh elasicseach and osquery? wazuh already covers a lot, but is this enough?

**Comment by weslambert** [comment, marked as answer]
It's all dependent on what you prefer/want. Both OSQ and Wazuh are optional and perform similar functions, but Elasticssearch is not (dependent on node role).

**Reply by 011248163264** [reply]
thanks
